administration/src/bp-modules/applications/PreVerifiedEntitlementType.tsx

```diff
@@ -0,0 +1,34 @@
+import { GeneralJsonField, type JsonField, findValue } from './JsonFieldView'
+
+export const enum PreVerifiedEntitlementType {
+  Juleica = 'blueCardJuleicaEntitlement',
+  Verein360 = 'blueCardWorkAtOrganizationsEntitlement',
+  HonoredByMinisterPresident = 'goldenCardHonoredByMinisterPresidentEntitlement',
+}
```

Comment:
🔧 Since enums are a bit problematic in Typescript, I would rather use an object with an associated type here:
```ts
export const preVerifiedEntitlements = {
  Juleica: 'blueCardJuleicaEntitlement',
  Verein360: 'blueCardWorkAtOrganizationsEntitlement',
  HonoredByMinisterPresident: 'goldenCardHonoredByMinisterPresidentEntitlement',
} as const
/** Values of {preVerifiedEntitlements} */
export type PreVerifiedEntitlementType = typeof preVerifiedEntitlements[keyof typeof preVerifiedEntitlements]
``` 

You can the refer to the actual values with e.g. `preVerifiedEntitlements.Juleica`.